modules/openy_features/openy_node/modules/openy_node_social_post/config/install/field.field.paragraph.social_list.field_prgrf_sl_description.yml

```diff
@@ -0,0 +1,25 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_prgrf_sl_description
+    - paragraphs.paragraphs_type.social_list
+  module:
+    - datalayer
+    - text
+third_party_settings:
+  datalayer:
+    expose: 0
+    label: field_prgrf_sl_description
+id: paragraph.social_list.field_prgrf_sl_description
+field_name: field_prgrf_sl_description
+entity_type: paragraph
+bundle: social_list
+label: Description
+description: ''
```

Comment:
Imagine the site will be used by babysitter and write a description of description field for her/him, please

Reply:
done.